packages/flame_3d/lib/src/components/light_component.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flame_3d/components.dart';
+import 'package:flame_3d/graphics.dart';
+import 'package:flame_3d/src/resources/light/light.dart';
+
+class LightComponent extends Component3D {
+  final Light light;
+
+  LightComponent({required this.light});
+
+  @override
+  void bind(GraphicsDevice device) {
+    world.device
+      ..model.setFrom(transformMatrix)
+      ..bindLight(light);
```

Comment:
This will grow the `lights` list in the graphics device on each frame right? We should clear the lights list in the `begin` method to ensure we dont have duplicate lights.

Reply:
oh good catch, I didn't realize that in my testing